### ISSUE TITLE
Print a collection of selected images or files

### ISSUE THREAD

**Issue by its-mXc** [elementary/files#1395]
There are lot of cases where  i need to print multiple files mostly images. A option to print the selected collection in context menu would be nice  :)

**Comment by jeremypw** [comment]
What is needed is a contractor file and a suitable application that can take multiple image files and print them given a suitable commandline.  Files is not intended to print files itself.  Photos can print multiple images but they all go into one document.  Is this what you want or is to batch print images as separate documents?

As Photos can already print images it may be possible to extend its commandline interface to print the files received as parameters.

**Comment by jeremypw** [comment]
There is a simple elementary app (https://github.com/elementary/print) for use with a contractor but as far as I can see it only accepts text files.  If you select multiple text files in Files you still get a "Print" contractor entry in the context menu.

**Comment by jeremypw** [comment]
I notice there is already an issue raised against io.elementary.print asking that it print images as well. See https://github.com/elementary/print/issues/3

**Comment by its-mXc** [comment]
Yes, i want the ability to print multiple images as separate documents and its interface directly from context menu

Also, thanks for the app recommendation, it will help me a lot :)

**Comment by its-mXc** [comment]
Ahh, i see it prints just one of the selected files : (

**Comment by jeremypw** [comment]
Closing as I think this extends the scope of Files too far and is best handled in Photos.
